examples/cross-contract-high-level/src/lib.rs

```diff
@@ -41,14 +41,15 @@ pub trait ExtStatusMessage {
 
 #[near_bindgen]
 impl CrossContract {
-    pub fn deploy_status_message(&self, account_id: AccountId, amount: U128) {
-        Promise::new(account_id)
+    pub fn deploy_status_message(&self, account_id: AccountId, amount: U128) -> Promise {
+        let new_account_id = format!("{}.{}", account_id, env::current_account_id())
+        Promise::new(new_account_id)
```

Comment:
```suggestion
    pub fn deploy_status_message(&self, mut account_id: AccountId, amount: U128) -> Promise {
        let new_account_id = format!("{}.{}", account_id, env::current_account_id());
        Promise::new(AccountId::new_unchecked(new_account_id))
```

I wonder if it's worth adding convenience functions to prepend or append `AccountId`s separated by a `.` so it doesn't require calling formatting like this.

Reply:
Hmm, I didn't need `AccountId::new_unchecked` here: https://github.com/near-examples/rust-high-level-cross-contract/pull/73/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759

I compiled and deployed the contract at that link and everything worked.

Reply:
how did you build? a string isn't an `AccountId` and the build fails on my machine. I'll fix and push the wasm change